install.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+
+# Authors:
+#   Sreeja Kamishetty <sreeja25kamishetty@gmail.com>
+
+# Description:
+#   A post-installation bash script for engelsystem
+
+echo "Update your package manager. $ sudo apt-get update."
+apt-get update
+
+echo "Installing LAMP"
+echo "Install Apache"
+apt-get install apache2
+
+echo "Install MySQL"
+apt-get install mysql-server php5-mysql
+
+echo "Install PHP"
+apt-get install php5 libapache2-mod-php5 php5-mcrypt
+
+echo "Install php cgi"
+apt-cache search php5-cgi
```

Comment:
Why PHP-CGI? Doesn't our web run on Apache's _mode_php_ (PHP embedded in Apache)?